var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -227,6 +229,9 @@ class Sirius(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("blis threads=openmp", when="+openmp ^blis")
     depends_on("intel-mkl threads=openmp", when="+openmp ^intel-mkl")
 
+    depends_on("wannier90", when="@7.5.0: +wannier90")
+    depends_on("wannier90+shared", when="@7.5.0: +wannier90+shared")
```

Comment:
Is the Wannier90 feature actually going to ship with `7.5.0`?

Reply:
Good point :). I was assuming the PR will get merged before the next release. @toxa81 This is the plan right?

Reply:
7.5.0 is not tagged yet and not in spack recipe (here); there are a few PRs that need to be merged including the work on Wannier90. To me `when="@7.5.0: +wannier90"` looks fine.